src/components/GettingStartedModal/ChooseSetStep.tsx

```diff
@@ -0,0 +1,111 @@
+import { ChangeEvent, FormEvent, ReactElement, useEffect , useState} from "react";
+import { NavigationButtons, Step, StepProps } from ".";
+import { Button } from "../Button";
+import {
+  isPhoneticsPreference,
+  PhoneticsPreference,
+  PREFERENCE_LITERATES,
+} from "../../state/reducers/phoneticsPreference";
+import { GROUPS, isGroupId } from "../../state/reducers/groupId";
+import { SectionHeading } from "../SectionHeading";
+import { CollectionDetails } from "../CollectionDetails";
+
+import { collections, VocabSet } from "../../data/vocabSets"; 
+import { useUserStateContext } from "../../providers/UserStateProvider";
+import { TermsByProficiencyLevelChart } from "../TermsByProficiencyLevelChart";
+
+export const ChooseSetStep: Step = {  
+  title: "Choose your first set",
+  commitState: ({ collectionId }, {setUpstreamCollection}) => {
+    if (collectionId !== undefined){
+      setUpstreamCollection(collectionId);
+    }
+  },
+  Component: PhoneticsStepComponent,
+};
+
+function PhoneticsStepComponent({
+  wizardState: { collectionId, groupId },
+  setWizardState,
+  goToNextStep,
+  goToPreviousStep,
+}: StepProps): ReactElement {
+  const [enabled, setEnbabled] = useState(collectionId != undefined); 
```

Comment:
1. I think it's best practice to use `!==` here.

2. Maybe "canGoToNextStep" would be a better name for this. It's not clear what is enabled/disabled by this until the last line of the function. Also, is this state, or a function of state? Ie. should `enabled` always be equal to `collectionId != undefined`? Is there any time when we want to have the button enabled even though the collectionId isn't set? If it's actually a function of another field and not its own data, you could remove the "useState" call and just do:
```
const canGoToNextStep = collectionId !== undefined;
```

Reply:
Yes this was a misunderstanding of state. I've changed this to use a global variable instead.